examples/fabric/cloud_router/go/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.21
 toolchain go1.22.6
 
 require (
-	github.com/equinix/pulumi-equinix/sdk 0.14.0+dirty
-	github.com/pulumi/pulumi/sdk/v3 v3.126.0
+	github.com/equinix/pulumi-equinix/sdk 0.15.0+dirty
```

Comment:
Where is 0.15.0 coming from?

Reply:
This may be related to [pulumi-equinix@0.15.0](https://github.com/equinix/pulumi-equinix/releases/tag/v0.15.0) and not pulumi-java, which also just went to 0.15.0.